tests/client.py

```diff
@@ -778,6 +780,103 @@ def create(self, stream, company_ids=[], subscriptions=[], times=1):
         else:
             raise NotImplementedError(f"There is no create_{stream} method in this dipatch!")
 
+    def create_custom_contact_properties(self):
+        """Create custom contact properties of all the types"""
+
+        url = f"{BASE_URL}/properties/v1/contacts/properties"
+        data = []
+        property = {
+            "name": "custom_string",
+            "label": "A New String Custom Property",
+            "description": "A new string property for you",
+            "groupName": "contactinformation",
+            "type": "string",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 6,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "custom_number",
+            "label": "A New Number Custom Property",
+            "description": "A new number property for you",
+            "groupName": "contactinformation",
+            "type": "number",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 7,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "custom_date",
+            "label": "A New Date Custom Property",
+            "description": "A new date property for you",
+            "groupName": "contactinformation",
+            "type": "date",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 9,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "custom_datetime",
+            "label": "A New Datetime Custom Property",
+            "description": "A new datetime property for you",
+            "groupName": "contactinformation",
+            "type": "datetime",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 10,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "multi_pick",
+            "label": "multi pick",
+            "description": "multi select picklist test",
+            "groupName": "contactinformation",
+            "type": "enumeration",
+            "fieldType": "checkbox",
+            "hidden": False,
+            "options": [
+              {
+                "label": "Option A",
+                "value": "option_a"
+              },
+              {
+                "label": "Option B",
+                "value": "option_b"
+              },
+              {
+                "label": "Option C",
+                "value": "option_c"
+              }
+            ],
+            "formField": True
+        }
+        data.append(deepcopy(property))
```

Comment:
It is great that you are creating a variety of data types in the custom fields.  Since you went through the pain of creating them we should also use them all when creating a contact - instead of just the first two - to make sure that we support all of the types.

Reply:
:) was expecting this comment.. The other types seems complicated, I'll need more time to figure out how to do. Can I write a separate card for that ?

Reply:
sure.